### PR TITLE
feat(south-rest-api): first poc

### PR DESCRIPTION
**Pull request by burgerni10**

### CODE REVIEW

File: src/south/RestApi/RestApi.class.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    this.apiType = apiType
+    this.host = host
+    this.port = port
+    this.entity = entity
+    this.authentication = authentication
+    this.connectionTimeout = connectionTimeout
+    this.requestTimeout = requestTimeout
+    this.startDate = startDate // "startDate" is currently a "hidden" parameter of oibus.json
+
+    const { engineConfig: { caching: { cacheFolder } } } = this.engine.configService.getConfig()
```
Comment (jfhenon) [inline src/south/RestApi/RestApi.class.js:41]: 41 -> 47. Maybe this code should be generic and push to the protocolHandler class 
this.tmpFolder = this.getTmpFolder()

**Review by jfhenon — APPROVED** [review]
one comment but not urgent.

**Review by jfhenon — APPROVED** [review]

File: src/south/RestApi/RestApi.class.js (hunk 1 above)
Comment (burgerni10) [reply src/south/RestApi/RestApi.class.js:69]: Gonna do this in another PR with bulk merged.

**Comment by burgerni10** [comment]
I've updated the branch from develop and added a commit to add queryParams and formatters to format the results

**Pull request merged**